docs/source/notes/multiprocessing.rst

```diff
@@ -27,9 +27,7 @@ others that require asynchronous operation.
 CUDA in multiprocessing
 -----------------------
 
-The CUDA runtime does not support the ``fork`` start method. However,
-:mod:`python:multiprocessing` in Python 2 can only create subprocesses using
-``fork``. So Python 3 and either ``spawn`` or ``forkserver`` start method are
+The CUDA runtime does not support the ``fork`` start method. In Python 3, either the ``spawn`` or ``forkserver`` start method are
```

Comment:
Does it sound a bit weird to say "in Python 3" here? Do you think we can simply remove the phrase?

Reply:
Removed reference to Python 3 here. Thanks